src/constructor.io/models/common/PageRequest.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections;
+using Constructorio_NET.Utils;
+
+namespace Constructorio_NET.Models
+{
+    /// <summary>
+    /// Constructor.io Pagination Request Class
```

Comment:
Can we update the comments to match the name change

Reply:
Whoops I totally missed that. Thanks for catching it!